tests/testthat/test-series_string.R

```diff
@@ -1,13 +0,0 @@
-test_that("Series to lower case", {
-  expect_identical(
-    pl$Series(c("A", "B", NA, "D"))$str$to_lowercase()$to_r(),
-    c("a", "b", NA, "d")
-  )
-})
-
```

Comment:
I don't think those should be removed. We probably don't need to test all methods for Series but at least a few of them to ensure that the dispatch from Expr to Series works fine